flink-kubernetes-operator-api/src/main/java/org/apache/flink/kubernetes/operator/api/status/CommonCRStatus.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.kubernetes.operator.api.status;
+
+import io.fabric8.kubernetes.api.model.Condition;
+import io.fabric8.kubernetes.api.model.ConditionBuilder;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/** Status of CR. */
+public class CommonCRStatus {
+
+    public static Condition crReadyTrueCondition(final String message) {
+        return crCondition("Ready", "True", message, "Ready");
+    }
+
+    public static Condition crReadyFalseCondition(final String message) {
+        return crCondition("Ready", "False", message, "Progressing");
+    }
+
+    public static Condition crErrorCondition(final String message) {
+        return crCondition("Error", "True", message, "UnhandledException");
+    }
+
+    public static Condition crCondition(
```

Comment:
I think if we rename the class we can also rename this methods simply to: `neady` / `notReady` / `error`

Reply:
Method names has been changed